.github/CODEOWNERS

```diff
@@ -8,99 +8,760 @@
 # Catch all
 /sdk/                                           @AlexGhiondea
 
-# Core
+# >>>>>>>>> Core Libraries <<<<<<<<<<
+
+# > Library: Azure.Core
```

Comment:
Is this format used for something? Or is it just for documentation?

Reply:
This is just a visual separator. I can remove if it causes issues or creates confusion!

Reply:
Changed to `#` 🙂.

Reply:
I think it might be worth removing as it might cause confusion. Without knowing all the details of the structure of this file I might expect this is required and maps to something somewhere when it is really just documentation, but is mostly duplicated information with other things so perhaps not worth having.